addons/wiki/models.py

```diff
@@ -180,11 +180,12 @@ def save(self, *args, **kwargs):
     def check_spam(self):
         request, user_id = get_request_and_user_id()
         user = OSFUser.load(user_id)
+        if not user:  # for tests and admin operations
+            return False
```

Comment:
FWIW, this should also turn into just a `return`. Also, still need to remove everything past L203